docs/applications/timemachine.md

```diff
@@ -0,0 +1,24 @@
+# Time Machine
+
+Apple docs: [https://support.apple.com/en-us/HT201250](https://support.apple.com/en-us/HT201250)
+Docker image: [https://github.com/mbentley/docker-timemachine](https://github.com/mbentley/docker-timemachine)
+
+Time Machine is an application that allows you to backup files from you Mac.
+
+## Usage
+
+Set `timemachine_enabled: true` in your `group_vars/all.yml` file.
+
+Set the Time Machine specific variables:
```

Comment:
All of this stuff should be under a "specific configuration" header - see https://github.com/davestephens/ansible-nas/blob/master/docs/applications/miniflux.md as an example.

Reply:
Done.